.github/labeler.yml

```
@@ -49,6 +49,28 @@
           - scripts/*.js
           - scripts/xcodebuild.sh
           - test/*.rb
+"platform: visionOS":
+  - changed-files:
+      - any-glob-to-any-file:
+          - .github/actions/cocoapods/*
+          - .github/actions/init-test-app/*
+          - .github/actions/setup-toolchain/*
+          - .github/workflows/build.yml
+          - Gemfile*
+          - ReactTestApp-DevSupport.podspec
+          - common/**/*
+          - example/visionos/**/*
+          - ios/*.rb
+          - ios/ReactTestApp/Manifest.swift
+          - ios/ReactTestApp/React+Compatibility.{h,m}
+          - ios/ReactTestApp/ReactInstance.swift
+          - ios/ReactTestApp/ReactTestApp-Bridging-Header.h
+          - ios/ReactTestApp/Session.swift
+          - ios/ReactTestApp/UIViewController+ReactTestApp.{h,m}
```

Looks like we're reusing almost everything:
```suggestion
          - ios/ReactTestApp/*.{h,m,mm,swift}
```